exercises/perfect-numbers/src/main/kotlin/NaturalNumber.kt

```diff
@@ -0,0 +1,8 @@
+
+enum class Classification {
+    DEFICIENT, PERFECT, ABUNDANT
+}
+
+fun getClassification(naturalNumber: Int): Classification {
```

Comment:
👍  `naturalNumber` is a _great_ name for this argument.  Little bits like this make exercises every so more pleasant to work on.

🤔 

`get` invokes the notion of an accessor in my head... but that's my Java talking. :)

Thinking something like `classify` might hit squarely?